cloudsql/mysql/database-sql/README.md

```diff
@@ -177,11 +177,28 @@ gcloud run deploy run-sql --image gcr.io/[YOUR_PROJECT_ID]/run-sql \
 
 Take note of the URL output at the end of the deployment process.
 
-
 Replace environment variables with the correct values for your Cloud SQL
 instance configuration.
 
-This step can be done as part of deployment but is separated for clarity.
+It is recommended to use the [Secret Manager integration](https://cloud.google.com/run/docs/configuring/secrets) for Cloud Run instead
+of using environment variables for the SQL configuration. The service injects the SQL credentials from
+Secret Manager at runtime via an environment variable.
+
+Create secrets via the command line:
+```sh
+echo -n "my-awesome-project:us-central1:my-cloud-sql-instance" | \
+    gcloud secrets versions add CLOUD_SQL_CONNECTION_NAME_SECRET --data-file=-
+```
+
+Deploy the service to Cloud Run specifying the env var name and secret name:
+```sh
+gcloud beta run deploy SERVICE --image gcr.io/[YOUR_PROJECT_ID]/run-sql \
```

Comment:
Why is this `beta`? At a quick glance, all these values are supported in the gcloud run command.

Reply:
the `--update-secrets` flag is in beta

Reply:
Oh. Got it. Thanks.